.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: php
+dist: trusty
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+    - $HOME/.cache/composer/files
+matrix:
+  include:
+    - php: 5.6
+  fast_finish: true
+before_script:
+  - composer global require squizlabs/php_codesniffer 
+  - composer global require phpunit/phpunit
```

Comment:
These should really be composer dev dependencies (at least phpunit) and we should adjust build.sh to use `./vendor/bin/phpunit` but happy to leave that for a separate PR

Reply:
+1 for separate PR, after #750 lands